CHANGELOG.md

```diff
@@ -5,7 +5,28 @@ All notable changes to OpenGHG will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased](https://github.com/openghg/openghg/compare/0.8.0...HEAD)
+## [Unreleased](https://github.com/openghg/openghg/compare/0.8.1...HEAD)
+
+## [0.8.1] - 2024-05-17
+
+### Added
+
+- Ability to convert from an old style NetCDF object store to the new Zarr based store format - [PR #967](https://github.com/openghg/openghg/pull/967)
+- Updated `parse_edgar` function to handle processing of v8.0 Edgar datasets. [PR #965](https://github.com/openghg/openghg/pull/965)
+- Argument `time_resolved` is added as phase 1 change for `high_time_resolution`, also metadata is updated and added deprecation warning. - [PR #968](https://github.com/openghg/openghg/pull/968)
+- Added explicit backwards compatability when searching previous object stores containing the `high_time_resolution` keyword rather than `time_resolved` - [PR #990](https://github.com/openghg/openghg/pull/990)
+- Added ability to pass additional tags as optional metadata through `standardising_*` and `transform_flux_data functions`. [PR #981](https://github.com/openghg/openghg/pull/981)
+
```

Comment:
Missing the following update from PR #952 

```
- More explanation regarding use of `search_*` and `get_*` function in tutorial 1 [PR #952](https://github.com/openghg/openghg/pull/952)
```

Reply:
This is in release 0.8.0

Reply:
Hmm this PR being merged is listed in the commit history for this branch which is why I highlighted it - any ideas why that might be as that should already be on main?

@SutarPrasad
[Merge pull request](https://github.com/openghg/openghg/pull/1011/commits/e28eff1d670b4f9d4d1d3fba86f28dcfff1f4615) https://github.com/openghg/openghg/pull/952 [from openghg/Iss338-search-and-get_obs_surfac…](https://github.com/openghg/openghg/pull/1011/commits/e28eff1d670b4f9d4d1d3fba86f28dcfff1f4615)
[e28eff1](https://github.com/openghg/openghg/pull/1011/commits/e28eff1d670b4f9d4d1d3fba86f28dcfff1f4615)

Perhaps the commit history has become a bit circular meaning the merge is showing up a few times.